docs/PROBLEM-SOLVING.md

```diff
@@ -0,0 +1,42 @@
+# Problem Solving Strategies
+
+
+As the first article here states, `Problem solving is a meta-skill`.
+We find that often, the syntax or structure of a programming language isn't the most difficult part of solving a challenge - figuring out _what to code_ or _where to start_ is.
+Developing strategies and a system to break down problems into code-able steps is a **great** skill to cultivate.
+Below are some community-sourced articles, videos, and books that can help you develop this super-skill.
+
+
+- Free Code Camp offers a good overview article on [How to Think Like a Programmer - Lessons in Problem-Solving][free-code-camp-think-like-a-programmer].
+- Kurtis Pykes writing for Towards Data Science has a nice [hands-on tutorial for problem-solving][Kurtis Pykes: Hands-on Tutorial - How to Improve Your Problem-Solving Skills as A Programmer].
+- UC Berkeley has a nice PDF summary of [G. Polya's Problem Solving Techniques][g-polya-how-to-solve-it-summary].
+- Originally written in 1945 as guidance for tackling complicated math problems,[G. Polya's How To Solve It][g-polya-how-to-solve-it] (full book) is still _excellent_ advice for problem-solving in general.
+- Mentioned in the Free Code Camp Article, V. Anton Spraul's [Think Like a Programmer: An Introduction to Creative Problem Solving][v-anton-spraul-think-like-a-programmer] is a more modern and programming-focused take on the same general methods Polya outlines for mathematics.
+- [Felienne Hermans][felienne-hermans] is more focused on _how_ people learn the art of coding and how that maps to learning in general.
+  She has written [The Programmers Brian][programmers-brain-free-online], with strategies for reading code better, thinking about code clearly, writing better code, and becoming a better code collaborator.
+  - You can order a physical copy of her book [here][programmers-brain-manning].
+  - She also has two interesting videos on the same topic: [How to Teach Programming (and other things??)][felienne-hermans-how-to-teach-programming] and [Writing is Programming is Writing][felienne-hermans-programming-is-writing-is-programming].
```

Comment:
```suggestion
  - She also has two interesting videos on the same topic: [How to Teach Programming (and other things??)][felienne-hermans-how-to-teach-programming] and [Programming is Writing is Programming ][felienne-hermans-programming-is-writing-is-programming].
```

Edited to account for the actual name given to the video on YouTube.

Reply:
🤦🏽‍♀️   _I got it exactly backward!_  Thank you so much for both checking and catching that!!  💙